web/tests/compose_pm_pill.test.cjs

```diff
@@ -81,21 +81,24 @@ run_test("pills", ({override, override_rewire}) => {
     let appendValue_called;
     pills.appendValue = function (value) {
         appendValue_called = true;
-        assert.equal(value, "othello@example.com");
+        assert.equal(value, othello.user_id.toString());
         this.appendValidatedData(othello);
     };
 
-    let get_by_email_called = false;
-    override_rewire(people, "get_by_email", (user_email) => {
-        get_by_email_called = true;
-        switch (user_email) {
-            case iago.email:
+    let maybe_get_user_by_id_called = false;
+    override_rewire(people, "maybe_get_user_by_id", (user_id, ignore_missing) => {
```

Comment:
Why do we need to rewire this? I feel like it should be much simpler to just set up the `people.ts` data as desired?.

Reply:
Okay, removed this.